clients/benchmarks/client.cpp

```diff
@@ -3,7 +3,8 @@
  * ************************************************************************ */
 
 #include <iostream>
-#include <stdio.h>
+#include <cstring>
```

Comment:
In C++11 and later, #include <cstdio> #include <cmath> #include <cstrings> etc. are preferred to #include <stdio.h> #include <math.h> #include <strings.h> etc.

Reply:
But you're replacing <stdio.h> with <cstring> as opposed to <cstdio>.  Is that what you intended?

Reply:
I will add cstdio, since printf is used below, even though it gets pulled in from other headers.